about.html

```diff
@@ -0,0 +1,99 @@
+                <li><a href="index.html" class="navigation-list" id="about-link">Home</a></li>
+                <li><a href="#" class="navigation-list">Program</a></li>
+                <li><a href="#" class="navigation-list">Join</a></li>
+                <li><a href="#" class="navigation-list">Sponsor</a></li>
+                <li><a href="#" class="button navigation-list">Summer Concert</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="main about-cover" id="background">
+        <div class="about-mid">
+            <h1 class="about-h1">Hello Piano world!</h1>
+            <h2 class="about-header">Piano CONCERT 2023</h2>
+            <p class="invitation-about">
+                Piano concert invition.Welcome to our concert. Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quidem, veniam enim? Corporis quisquam quos obcaecati minus consectetur ratione iusto veritatis nam, illo quibusdam itaque aut, dolorum unde culpa
+                delectus. Necessitatibus totam, esse delectus doloribus repellendus enim illum exercitationem rerum dolorem dolor voluptate amet? Placeat adipisci ducimus, molestias illum aperiam provident?</p>
+
+            <p class="contact-us">
+                Do you have questions about our concert Event? </p>
+            <a href="#" class="email">hsuwady2001@gmail.com </a>
+        </div>
+    </section>
+    <section class="section-2">
+        <h2 class="about-h2">Piano Concert 2023 Event</h2>
+        <div class="bottom-line"></div>
+        <p class="  about-sponser">
+            Delve into the captivating stories from our previous summer concerts held in diverse locations. Experience the mesmerizing melodies that filled the air, creating an enchanting ambiance for all attendees.
+            <br><br> These extraordinary events showcased the power of music and unity within our own local community. Immerse yourself in the inspiring narratives of these memorable gatherings, where the spirit of music and the joy of cultural exchange
+            seamlessly intertwined.
+        </p>
+        <img src="images/piano logo.png" alt="logo-2">
+    </section>
+
+    <section class="section-3">
+        <h2 class="about-h2">Discover previous piano concerts</h2>
+        <div class="bottom-line"></div>
+        <p class="about-sponser">
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptatum sint quas fuga dignissimos expedita impedit possimus deserunt hic in facere?
+            <br><br> Lorem ipsum dolor, sit amet consectetur adipisicing elit. Repellendus sapiente facilis beatae praesentium quia maiores! Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nisi nesciunt et delectus, doloremque minus perspiciatis.
+        </p>
+
+        <div class="past-projects">
+            <div class="project-1 project">
+            </div>
+            <div class="project-2 project">
+            </div>
+        </div>
```

Comment:
- [ ] Please stick to the original design by Cindy and implement the following features. Remember that you need to stick to the colors, fonts, ... You can change the words & images to suit your theme. 😃 
-  The images in this section have text centered over them.
-  The images also have a transparent layer of red color over them

**Cindy design**
![img1](https://github.com/Hsuwaddyhtat22/Microverse-first-capstone-project/assets/99656687/7986f70d-1402-4e32-8824-2fa78697b714)

**Your implementation**
![img2](https://github.com/Hsuwaddyhtat22/Microverse-first-capstone-project/assets/99656687/24c1a589-b771-491d-ad11-3e6188b112e2)